src/plugins/discover/public/application/components/types.ts

```diff
@@ -159,23 +159,8 @@ export interface DiscoverProps {
    */
   timeRange?: { from: string; to: string };
   /**
-   * An object containing properties for proper handling of unmapped fields in the UI
+   * callback function to change the value of `showUnmappedFields` flag
+   * @param value new value to set
    */
-  unmappedFieldsConfig?: {
```

Comment:
Why the need to change this now? If we're planning to phase-out the unmapped fields over the next few minors, this config value will have its purpose and it doesn't really affect things atm.